.github/workflows/release-integration.yml

```diff
@@ -3,6 +3,7 @@ name: Release Integration Environment
 on:
   push:
     branches: [ main ]
+  workflow_dispatch:
```

Comment:
I see what you mean, yes, am not sure 😅.
but I think it should work, since this is meant to be triggered on any push to `main` so, it has to work over and over with the same identifier.